app/main.py

```
@@ -0,0 +1,49 @@
+        await conn.run_sync(models.Base.metadata.create_all)
+
+
+# Get all ToDo items
+@app.get("/todos", response_model=list[schemas.ToDoResponse])
+async def read_todos(db: AsyncSession = Depends(get_db)):
+    return await crud.get_todos(db)
+
+
+# Get a single ToDo item
+@app.get("/todos/{todo_id}", response_model=schemas.ToDoResponse)
+async def read_todo(todo_id: int, db: AsyncSession = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `read_todo`

(ANN201)